lib/membrane/rtcp/packet_parser.ex

```diff
@@ -0,0 +1,100 @@
+defmodule Membrane.RTCP.PacketParser do
```

Comment:
Would call it just Packet

Reply:
OK, that would solve naming conflict with parser element :+1: